src/components/NcCounterBubble/NcCounterBubble.vue

```diff
@@ -55,6 +155,29 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+
+		/**
+		 * The count to display in the counter bubble. String value is parsed to a number.
+		 * Alternatively, you can pass any value to the default slot.
+		 */
+		count: {
+			type: [Number, String],
+			required: false,
+			default: 0,
+		},
+
+		/**
+		 * The maximal count number to be display. If the count is higher then "max+" is displayed.
+		 * For example, with limit 999, 1000 will be displayed as "999+".
+		 * Only works if the count is set via the `count` prop and not via the default slot.
+		 * String value is parsed to a number.
+		 * Set 0 to disable the limit.
+		 */
+		limit: {
+			type: [Number, String],
```

Comment:
why allow string? I mean I am fine with that as it easily allows `limit="99"` but I am curious if there is more behind that decision.

Reply:
To make it easier to pass constants. But I guess, as it is always passed from data, it should be number anyway. So, it can be removed.

Reply:
Removed